Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN set -x \
   /var/lib/dpkg/info/ca-certificates-java.postinst configure \
   # Install npm dependencies
   && cd /dcos-ui \
-  && npm install -g  git://github.com/johntron/http-server.git#proxy-secure-flag \
+  && npm install -g junit-merge git://github.com/johntron/http-server.git#proxy-secure-flag \
```

Comment:
maybe we should add a specific version here? otherwise it _could_ break after an update and block our CI…